tests/garage/logger/test_snapshotter.py

```diff
@@ -0,0 +1,69 @@
+from os import path as osp
+import pickle
+import random
+import shutil
+import unittest
+
+from garage.logger import snapshotter
+from garage.misc.console import mkdir_p
+
+
+class TestSnapshotter(unittest.TestCase):
```

Comment:
Thanks for the test! Though I thought we agreed you didn't have to write tests for the snapshotter?

Reply:
Lol I refactored a bit so I just did it; didn't take too long